.github/workflows/post-release.yaml

```diff
@@ -54,7 +54,6 @@ jobs:
   update-docs-version:
     name: Update docs version
     runs-on: ubuntu-latest
-    environment: post-release
```

Comment:
I don't think this should be removed. It will break `${{ vars.APP_ID }}` and `${{ secrets.PRIVATE_KEY }}` used in the `generate_token` step.

Reply:
Are you sure? I thought it just made the run show up in "Deployments" and this is not really a deployment.

Reply:
Yep, I'm sure. `${{ vars.XXX }}` only makes sense in the context of an environment, and the secret also comes from the `post-release` environment: https://github.com/gravitational/github-terraform/blob/996bb20e2bbed7740e825defc2b6e1ba13018b19/gravitational/github-actions-values.tf#L86.

I added this environment in #30890 to make this work.

It's annoying that they have attached the "Deployment" terminology to environments when it does not necessarily have anything to do with deploying. I guess that's the main use case.

Reply:
Thanks for the explanation and for catching it!